src/herder/Upgrades.cpp

```diff
@@ -74,6 +74,24 @@ Upgrades::applyTo(LedgerUpgrade const& upgrade, LedgerHeader& header)
     }
 }
 
+std::string
+Upgrades::toString(LedgerUpgrade const& upgrade)
+{
+    switch (upgrade.type())
+    {
+    case LEDGER_UPGRADE_VERSION:
+        return fmt::format("VERSION={0}", upgrade.newLedgerVersion());
```

Comment:
I think we now use `PROTOCOL_VERSION` now that the string is going to be used in various context (so that it matches what we use in Config for all fields)